crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -123,6 +123,11 @@ module deepbook::clob_v2 {
         price: u64
     }
 
+    /// Emitted when all orders are canceled.
+    struct AllOrdersCanceled<phantom BaseAsset, phantom QuoteAsset> has copy, store, drop {
+        orders_canceled: vector<OrderCanceled<BaseAsset, QuoteAsset>>,
```

Comment:
It's definitely most convenient to just have a vector of the existing event type. But I think we can save space by pulling `pool_id` and `owner` into fields of `AllOrdersCanceled` and then define a new type with the fields that are unique for each order. That's potentially a big space savings when there are a lot of events--e.g., 256 * 64 ~= 16KB when there are 256 events.

Reply:
The `pool_id` can go there but the orders could be owned by other people right, so owner can't be added to this struct.

Reply:
Yep, good point! I forgot that Alice can cancel Bob's orders.